src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService service;
+        service = new UserService();
+        return service.findByEmail(email) && service.findByPassword(password);
```

Comment:
we can just compare the password we got as a method parameter and found user's password